crates/tests/winrt/tests/composition.rs

```diff
@@ -68,8 +69,8 @@ fn composition() -> windows::core::Result<()> {
 
     // Visual.set_brush expects a CompositionBrush but CreateColorBrushWithColor returns a
     // CompositionColorBrush that logically derives from CompositionBrush.
-    let brush = compositor.CreateColorBrushWithColor(&red)?;
-    visual.SetBrush(brush)?;
+    let brush = compositor.CreateColorBrushWithColor(red)?;
+    visual.SetBrush(&CompositionBrush::from(brush))?;
```

Comment:
This is unfortunate. We should consider what it would take to make this work for class hierarchies like this. This is a case where the parameter always expects a class (not an interface) and there is a closed set of possibilites. This is somewhat like exclusive interfaces.